pkg/controller/extension/constants.go

```diff
@@ -0,0 +1,9 @@
+package extension
+
+import fluxv1alpha1 "github.com/stackitcloud/gardener-extension-shoot-flux/pkg/apis/flux/v1alpha1"
+
+const (
+	managedByLabelKey      = "app.kubernetes.io/managed-by"
+	managedByLabelValue    = "gardener-extension-" + fluxv1alpha1.ExtensionType
+	shootInfoConfigMapName = "shoot-info-envsubst"
```

Comment:
`envsubst` seems a bit colloquial to me. Should we go for something more formal like `shoot-info-substitutions` or `shoot-info-env-substitutions`?

Reply:
At the moment I'm thinking about naming it just `shoot-info`.

Reply:
Sounds good to me